### DISCUSSION TITLE
Can't make private repo publicly visible

### DISCUSSION THREAD

**Discussion by cybermaggedon** [community/community#54268] [Repositories]
### Select Topic Area

Bug

### Body

I tried to change visibility, and I just get an error:
```
Sorry, repository visibility cannot be changed at this time. A previous visibility change is still in progress.  
```
The repo belongs to an organisation.  I searched, can't see anyone talking about this error before. Tried again 30 mins later, still seeing this error.

**Comment by cybermaggedon** [comment]
Intermittent. Went away, or been fixed.

**Reply by zczvt64ydz-tech** [reply]
![image](https://github.com/user-attachments/assets/d29937ce-3b9b-4ccf-a192-95acf7270194)

**Comment by zczvt64ydz-tech** [comment]
![image](https://github.com/user

-attach@**

**ments/assets/e8ce3d4e-6593-4e3c-96fb-780478ce2269)
